bin/specextract

```diff
@@ -1689,7 +1775,7 @@ def groupspec(args):
             raise IOError(f"Failed to group spectrum for {fullfile}")
```

Comment:
I know this line want's changed, but `fullfile` isn't a defined variable in this routine.

Reply:
It's also an issue for line 1787.

Reply:
And it would pobably make sense to change these from

```
    except OSError:
        raise IOError(..)
```

to

```
    except OSError as err:
        raise IOError(...) from err
```

as you get a slightly-more-helpful error trace if they do fail.